package.json

```diff
@@ -44,6 +44,9 @@
         "subPermissions": [
           "remote-storage.configurations.collection.get",
           "remote-storage.configurations.item.get",
+          "remote-storage.configurations.item.post",
```

Comment:
.all?

Reply:
Do you mean `remote-storage.all`?
That would be too much.
https://github.com/folio-org/mod-remote-storage/blob/6af6afc9e5c7d8bf68b573ec02622b385d2abe30/descriptors/ModuleDescriptor-template.json#L316